src/test/java/edu/ucsb/cs156/courses/controllers/PersonalCoursesControllerTests.java

```diff
@@ -85,6 +97,31 @@ public void api_schedules_admin_all__admin_logged_in__returns_all_schedules() th
         String responseString = response.getResponse().getContentAsString();
         assertEquals(expectedJson, responseString);
     }
+    @WithMockUser(roles = { "USER" })
+    @Test
+    public void api_courses__admin_logged_in__get_courses_by_personal_schedule_ID() throws Exception {
+
+
+        PersonalCourses p1 = PersonalCourses.builder().psId(1).enrollCd("123456").quarter("20222").id(0L).build();
+        PersonalCourses p2 = PersonalCourses.builder().psId(1).enrollCd("789123").quarter("20222").id(1L).build();
+        PersonalCourses p3 = PersonalCourses.builder().psId(2).enrollCd("654321").quarter("20222").id(2L).build();
```

Comment:
This test is testing that, if the endpoint is called with `psID = 1`, that all `psID = 1` is returned. This personal course has `psID = 2`, so it probably shouldn't be returned...